laravel/resources/views/pages/applications/list.blade.php

```diff
@@ -8,8 +8,8 @@
     </h1>
     @foreach($rounds as $round)
         <hr>
-        <h2> {{ $round->name }} </h2>
-        <div class="scrollable">
+        <h2 class="appTitle"> {{ $round->name }} </h2>
+        <div class="scrollable hidden">
```

Comment:
@Meleeman01 As per the requirements in the original issue #63, only rounds that are older than a year should have their applications hidden by default.

Reply:
aight I got that in there. I used Carbon's isLastYear() function and it seems to do it only if the date is from last year and not if the date is exactly 1 year. not sure which one you wanted but I can change it if you like.